docs/installation/base-url.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Add ability to run Headlamp with a base-url
```

Comment:
This should be called: `Run Headlamp with a custom base URL` or something like that. Otherwise it looks like a PR :slightly_smiling_face: 
![Screenshot from 2021-02-25 14-25-20](https://user-images.githubusercontent.com/1029635/109167557-c9799700-7775-11eb-9730-0a3b2f795ec8.png)

Reply:
Fixed, thanks.